src/main/java/org/cascadebot/cascadebot/commands/developer/DumpCommand.java

```diff
@@ -26,33 +26,33 @@
     @Override
     public void onCommand(Member sender, CommandContext context) {
         if (context.getArgs().length < 1) {
-            context.getTypedMessaging().replyDanger("Hmmm either pick: `threads`, `commands`, `permissions` or `guild`");
+            context.getTypedMessaging().replyDanger(context.i18n("commands.developer.dump.not_enough_arguments"));
```

Comment:
do we really need to languageify developer commands?

Reply:
No I don't think we should tbh